lib/graphql/schema/printer.rb

```diff
@@ -132,16 +132,22 @@ def print_input_value(arg)
           def print_value(value, type)
             case type
             when FLOAT_TYPE
+              return 'null' if value.nil?
```

Comment:
This makes me a bit sad, maybe this method is doing too much. It's both validating `type` and coercing `value` to IDL.